tests/ga/test_update.py

```diff
@@ -1648,8 +1648,8 @@ def _mock_popen(cmd, *args, **kwargs):
                 with patch('azurelinuxagent.common.conf.enable_firewall', return_value=True):
                     update_handler.run(debug=True)
 
-        # Firewall-cmd should only be called 3 times - 1st to check if running, 2nd & 3rd for the QueryPassThrough cmd
-        self.assertEqual(3, len(executed_commands), "The number of times firwall-cmd should be called is only 3")
+        # Firewall-cmd should only be called 4 times - 1st to check if running, 2nd & 3rd & 4th for the QueryPassThrough cmd
+        self.assertEqual(3, len(executed_commands), "The number of times firwall-cmd should be called is only 4")
```

Comment:
This test is not doing anything as of now, can you please revert the comments here

Reply:
Fixed the test according to new logic